kopf/__init__.py

```diff
@@ -20,7 +20,10 @@
     LOGLEVEL_ERROR,
     LOGLEVEL_CRITICAL,
     EventsConfig,
-    WorkersConfig
+    WorkersConfig,
+)
+from kopf.engines.logging import (
+    ObjectLogger,
```

Comment:
Why export this type at such a high level? It is not used from the root `kopf` module internally and, if I understand correctly, the only interactions the user could/should have with it are:
- use the instance already provided as the `logger` parameter in handlers (so no need to call the constructor),
- in type annotations, where encouraging users to rather use the _Logger_  or _LoggerAdapter_ interfaces would give Kopf more freedom for future refactorings.

Reply:
You are right, let's not expose it to the public interface until there is a use-case. It is highly doubtful if anyone will instantiate their own per-object loggers at all. And it depend on the event-posting asyncio queue instance, which is surely not available to the users (it is part of Kopf's internals).

I will remove it from here.

Reply:
Removed.